tdp_vars_defaults/kafka/kafka.yml

```diff
@@ -157,7 +157,8 @@ jmxremote_password: Tdpjmx123,
 jmx_exporter_root_dir: /opt/tdp/jmx-exporter
 jmx_exporter_install_file: "{{ jmx_exporter_root_dir }}/jmx-exporter.jar"
 jmx_common_opts: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=true -Dcom.sun.management.jmxremote.ssl=true"
-jmx_exporter_kafka_opts: "{% if 'exporter_jmx' in groups and groups['exporter_jmx'] %}-javaagent:{{ jmx_exporter_install_file }}=18120:{{ jmx_exporter_conf_dir }}/kafka.yml{% else %}{% endif %}"
+jmx_exporter_kafka_port: 18202
```

Comment:
Can you use the same name pattern and move it in `tdp-cluster` file ?

You can see a name pattern example here: https://github.com/TOSIT-IO/tdp-collection/blob/8c1a55ba6b8dbad10646148e16905c659eccb50b/tdp_vars_defaults/tdp-cluster/tdp-cluster.yml#L94

You can see the usage of this variable here: https://github.com/TOSIT-IO/tdp-collection/blob/8c1a55ba6b8dbad10646148e16905c659eccb50b/tdp_vars_defaults/hadoop/hadoop.yml#L47

Reply:
There is no tdp_vars_default/tdp-cluster/tdp-cluster.yml file in this repository. Should I create it ?

Reply:
Yes you can create it.

Reply:
Done.